<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iwop4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_Programmliste">
     <attribute name="title">
      <string>Programme</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_Programmliste">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>501</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Werkzeug">
     <attribute name="title">
      <string>Werkzeug</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_Werkzeug">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeu"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionProgrammliste_anzeigen"/>
   </widget>
   <widget class="QMenu" name="menuProduzieren">
    <property name="title">
     <string>Produzieren</string>
    </property>
    <addaction name="actionMakeProgrammkopf"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuExtras"/>
   <addaction name="menuProduzieren"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNeu">
   <property name="text">
    <string>Neu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionProgrammliste_anzeigen">
   <property name="text">
    <string>Programmliste anzeigen</string>
   </property>
  </action>
  <action name="actionMakeProgrammkopf">
   <property name="text">
    <string>Programmkopf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
